src/pages/[_network]/accounts/[_id].tsx

```diff
@@ -186,21 +191,14 @@ const AccountPage: NextPage = () => {
                       secondary="Address"
                       primary={
                         accountQuery.data ? (
-                          <Tooltip title="View on blockchain explorer">
-                            <AppLink
-                              href={network.getLinkForAddress(
+                          <div>
```

Comment:
Super small thing but I would have used `<></>` here because I believe this data is already wrapped by a `div` so putting one ourselves does not add to the semantics.

Reply:
Good point, I will remove it ;)